cmd/cluster/powervs/create.go

```diff
@@ -0,0 +1,172 @@
+package powervs
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/signal"
+	"syscall"
+
+	apifixtures "github.com/openshift/hypershift/api/fixtures"
+	"github.com/openshift/hypershift/cmd/cluster/core"
+	powervsinfra "github.com/openshift/hypershift/cmd/infra/powervs"
+	"github.com/openshift/hypershift/cmd/log"
+	"github.com/openshift/hypershift/support/infraid"
+	"github.com/spf13/cobra"
+)
+
+const (
+	defaultCIDRBlock = "10.0.0.0/16"
+)
+
+func NewCreateCommand(opts *core.CreateOptions) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:          "powervs",
+		Short:        "Creates basic functional HostedCluster resources on PowerVS PowerVS",
+		SilenceUsage: true,
+	}
+
+	opts.PowerVSPlatform = core.PowerVSPlatformOptions{
+		APIKey:        os.Getenv("IBMCLOUD_API_KEY"),
+		PowerVSRegion: "us-south",
+		PowerVSZone:   "us-south",
+		VpcRegion:     "us-south",
+		SysType:       "s922",
+		ProcType:      "shared",
+		Processors:    "0.5",
+		Memory:        "32",
+	}
+
+	cmd.Flags().StringVar(&opts.PowerVSPlatform.ResourceGroup, "resource-group", "", "IBM Cloud Resource group")
+	cmd.Flags().StringVar(&opts.PowerVSPlatform.PowerVSRegion, "powervs-region", opts.PowerVSPlatform.PowerVSRegion, "IBM Cloud PowerVS region")
```

Comment:
none of this needs the "powervs-" prefix since this is the powervs subcommand already.